docs/docs/modules/agents/agent_types/openai_assistant.mdx

```diff
@@ -0,0 +1,190 @@
+# OpenAI Assistant
+
+:::info
+The [OpenAI Assistant API](https://platform.openai.com/docs/assistants/overview) is still in beta. OpenAI does not recommend it for production use.
+:::
+
+OpenAI released a new API for a conversational agent like system called Assistant.
+It's similar to LangChain's AgentExecutor, but includes some other features like a stateful API, integrated tools ([code interpreter](https://platform.openai.com/docs/assistants/tools/code-interpreter), [retrieval])(https://platform.openai.com/docs/assistants/tools/knowledge-retrieval) and the ability to pass custom tools (similar to [OpenAI functions](https://js.langchain.com/docs/modules/agents/agent_types/openai_functions_agent)).
+
+We've implemented the assistant API in LangChain with some helpful abstractions. In this guide we'll go over those, and show how to use them to create powerful assistants.
```

Comment:
Add something like:
```
Add something like:
```
You can interact with OpenAI Assistants using OpenAI tools or custom tools. When using exclusively OpenAI tools, you can just invoke the assistant directly and get final answers. When using custom tools, you can run the assistant and tool execution loop using the built-in AgentExecutor or write your own executor.
```